src/main/java/frc/robot/RobotContainer.java

```diff
@@ -132,10 +132,10 @@ public RobotContainer() {
    * joysticks}.
    */
   private void configureBindings() {
-    OI.getButton(OI.Operator.A)
-        .onTrue(
-            new InstantCommand(robotStateManager::switchPlacementMode)
-                .withName("Switch Placement Mode Command"));
+    // OI.getButton(OI.Operator.A)
```

Comment:
Why is this commented out? If we will likely need to turn it back on, you should add a comment stating that. If not, it should be removed.

Reply:
IDK why this was commented out. I uncommented it.